app/controllers/api/deploy_groups_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class Api::DeployGroupsController < Api::BaseController
+  include DeployGroupsCommon
+  before_action :enabled?
+
+  def index
+    scope = current_project.present? ? stage.deploy_groups : deploy_groups
```

Comment:
yeah

On Tue, Aug 16, 2016 at 5:15 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/controllers/api/deploy_groups_controller.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r75044310>:
>
> > @@ -0,0 +1,22 @@
> > +# frozen_string_literal: true
> > +class Api::DeployGroupsController < Api::BaseController
> > +  include DeployGroupsCommon
> > +  before_action :enabled?
> > +
> > +  def index
> > +    scope = current_project.present? ? stage.deploy_groups : deploy_groups
>
> so just nix the concern?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/66abc156d3f3ad8478ca07ee5c0e76ed494978bf#r75044310>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ7mfyXv4UAHURB7-DypoHmNfjbMZks5qglKwgaJpZM4Jhtog>
> .
>